What's-new-in-TypeScript.md

```diff
@@ -78,7 +84,7 @@ function f(x) { // Error: Not all code paths return a value.
 }
 ```
 
-### Case labels fall-through 
+### Case clause fall-throughs
 
 Reports errors for fall-through cases in switch statement. Check is turned *off* by default, and can enabled using `--noFallthroughCasesInSwitch`.
```

Comment:
Is there a way to temporarily disable the check, like `// FALLTHROUGH` or something?